includes/alerts.inc.php

```diff
@@ -160,13 +160,14 @@ function RunRules($device)
         return false;
     }
     foreach (GetRules($device) as $rule) {
-        echo " #".$rule['id'].":";
+        c_echo('Rule %p#'.$rule['id'].' (' . $rule['name'] . '):%n ');
         $inv = json_decode($rule['extra'], true);
         if (isset($inv['invert'])) {
             $inv = (bool) $inv['invert'];
         } else {
             $inv = false;
         }
+        d_echo(PHP_EOL);
```

Comment:
I noticed all the other calls use c_echo without the second argument.

If you want this output for all poller runs,  you should change this to just echo.  Otherwise, add $debug to the c_echo calls.